docs/pages/connect-your-client/introduction.mdx

```diff
@@ -19,7 +19,6 @@ your Teleport cluster:
 
 <Tabs>
 <TabItem label="Local user">
-
```

Comment:
Code blocks should have newlines around them.

Reply:
Since when? There's at least 50 examples of this _not_ being the case, so if we actually care about this it should be enforced by a linter.